events/recorder_histogram_grouped.go

```diff
@@ -0,0 +1,77 @@
+package events
```

Comment:
Could you point me to where this code differs from the `histogramStream` implementation?